packages/react-lowcode/src/templates/index.ts

```diff
@@ -0,0 +1,49 @@
+export const entityFieldMuiMap = `
+(eF) => {
+  return {
+    field: eF.name,
+    flex: 1,
+    type: eF.type,
+    valueFormatter: getValueFormatterFunction(eF.type),
+    renderHeader: (params: GridColumnHeaderParams) => (
+      <FormattedMessage id={\`\${eF.name}.\${eF.type}\`} defaultMessage={eF.name} />)
+  }
+}
+`
+
+export const entityFieldGrommetMap = `
+(eF) => {
+  return {
+    property: eF.name,
+    header: <FormattedMessage id={\`\${eF.name}.\${eF.type}\`} defaultMessage={eF.name} />,
```

Comment:
you need import everything (please move the string tempalates to code that is validated by vsdoce/typescript/linter)